README.md

```diff
@@ -375,10 +393,10 @@ Usage is identical to the numeric-query command:
 Complete argument list:
 
     scalyr timeseries-query [filter] [--function xxx] --start xxx [options...]
-        Just like numeric-query, but with Scalyr creating a timeseries for you in the background
+        Just like numeric-query, but with Scalyr creating a timeseries for you in the background, unless `createSummaries = false`
 
     scalyr timeseries-query --timeseries <timeseriesid> --start xxx [options...]
-        To query a timeseries created using create-timeseries
+        To query a timeseries created using create-timeseries 
```

Comment:
What is `create-timeseries` and is it ever used? And where is the --timeseries flag parsed, if at all?

Reply:
`create-timeseries` is a (deprecated) call that would create a timeseries and return an ID, which ID you would then use to query that timeseries.  We got rid of that in favor of 'just use the query string, no need for an ID' many years ago.  At this point we could remove `create-timeseries` as well as the `--timeseries` flag from the tool, we marked them as deprecated (in the README) years ago.  Different PR

Reply:
Ok cool! I will create a new ticket and PR to remove references to `create-timeseries` and `--timeseries`.

Reply:
FRONT-6857 Remove references to `create-timeseries` in scalyr-tool